### DISCUSSION TITLE
Change time interval via variable?

### DISCUSSION THREAD

**Discussion by radoeka** [grafana/grafana-csv-datasource#123] [Q&A] [answered]
With other data sources it is possible to alter the time interval using a variable.

The interval could be changed to for example 1 day, 1 week, 2 week, 1 month.

Is this possible with Grafana cvs plugin?  If not, will it be made available?

**Comment by marcusolsson** [comment, marked as answer]
Unfortunately, there's currently no support for filtering, though it sounds like something that would be possible with #58.
